feature_layers/feature-collection-layer-from-portal/src/main/java/com/esri/samples/feature_collection_layer_from_portal/FeatureCollectionLayerFromPortalSample.java

```diff
@@ -0,0 +1,151 @@
+import javafx.scene.control.TextField;
+import javafx.application.Application;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+
+public class FeatureCollectionLayerFromPortalSample extends Application {
+
+    private MapView mapView;
+    private ArcGISMap map;
+    private Portal portal;
+    private FeatureCollectionLayer featureCollectionLayer;
+    private FeatureCollection featureCollection;
+    private TextField input;
+
+    @Override
+    public void start(Stage stage) {
+        // set up the scene
+        try {
+            // create stack pane and application scene
+            StackPane stackPane = new StackPane();
+            Scene scene = new Scene(stackPane);
+
+            // set title, size, and add scene to stage
+            stage.setTitle("Feature Collection Layer From Portal");
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(scene);
+            stage.show();
+
+            // create amp and set it to be displayed in this view
+            map = new ArcGISMap(Basemap.createOceans());
+            // create a view for this ArcGISMap and set ArcGISMap to it
+            mapView = new MapView();
+            mapView.setMap(map);
+
+            // create text field to input user's own coordinate string
+            input = new TextField();
+            input.setMaxWidth(250);
+            input.setText("32798dfad17942858d5eef82ee802f0b");
+
+            // create button to perform action
+            Button fetchFromPortal = new Button("Open from portal item");
+            fetchFromPortal.setMaxWidth(250);
+
+            /**
+             * Action event on button and validation of empty Text Filed
+             */
+            fetchFromPortal.setOnAction(e -> {
+
+                if (!"".equals(input.getText())) {
+                    fetchFeatureCollectionFromPortal();
+                } else {
+                    new Alert(Alert.AlertType.ERROR, "Portal Itemid is empty. Please enter a portal item id.").show();
+                }
+
+            });
+
+            fetchFeatureCollectionFromPortal();
+
+            // add feature layer to ArcGISMap
+            map.getOperationalLayers().add(featureCollectionLayer);
+
+            // add the map view and control panel to stack pane
+            stackPane.getChildren().addAll(mapView, input, fetchFromPortal);
+            StackPane.setAlignment(input, Pos.TOP_LEFT);
+            StackPane.setMargin(input, new Insets(10, 0, 0, 10));
+            StackPane.setAlignment(fetchFromPortal, Pos.TOP_LEFT);
+            StackPane.setMargin(fetchFromPortal, new Insets(40, 0, 0, 10));
+
+        } catch (Exception e) {
+            // on any error, display the stack trace
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Fetch feature collection from portal and load as feature layer
+     */
+    private void fetchFeatureCollectionFromPortal() {
+
+        //create portal and portal item
```

Comment:
Missing a space here before the comment:
```suggestion
        // create portal and portal item
```